source/RialtoGStreamerMSEAudioSinkPrivate.h

```diff
@@ -38,6 +47,10 @@ struct _RialtoMSEAudioSinkPrivate
     std::atomic_bool isSyncOffQueued = false;
     std::atomic<int32_t> streamSyncMode = kDefaultStreamSyncMode;
     std::atomic_bool isStreamSyncModeQueued = false;
+    std::atomic<const gchar*> audioFade = kDefaultAudioFade;
+    AudioFadeConfig audioFadeConfig;
+    std::atomic_bool isAudioFadeQueued = false;
+    std::atomic<uint32_t> fadeVolume = kDefaultFadeVolume;
```

Comment:
Audiofade in priv is never set, but we dont set it. So we can remove it and just return the default volume on get failure.